src/cljs/todo_app_two/core.cljs

```diff
@@ -39,13 +39,40 @@
   [:section.section>div.container>div.content
    [:img {:src "/img/warning_clojure.png"}]])
 
+(defn create-incompleted-todos
```

Comment:
Hierdie en completed is die selle

Reply:
Also dis nie "create" nie, dis 'n show, en die namespace is klaar vir views.
So die function kan net `todo` wees